landcover.mss

```diff
@@ -546,13 +527,18 @@
 }
 
 #landuse-overlay {
-  [landuse = 'military'][zoom >= 10]::landuse {
+  [feature = 'military'][zoom >= 10] {
```

Comment:
I'm wondering if I should name the column `landuse`, I'm just normalizing the two different wood taggings into one rather than a full feature column setup.